fri:
  port: 30200
  robot_ip: 192.170.10.2
  robot_description: /robot_description

hardware_interface:
  control_freq: 200 # in Hz
  joints:
    - $(arg robot_name)_joint_1
    - $(arg robot_name)_joint_2
    - $(arg robot_name)_joint_3
    - $(arg robot_name)_joint_4
    - $(arg robot_name)_joint_5
    - $(arg robot_name)_joint_6
    - $(arg robot_name)_joint_7
